src/types.db

```diff
@@ -51,13 +51,16 @@ df                      used:GAUGE:0:1125899906842623, free:GAUGE:0:112589990684
 df_complex              value:GAUGE:0:U
 df_inodes               value:GAUGE:0:U
 dilution_of_precision   value:GAUGE:0:U
+disk_allocation         value:DERIVE:0:U
```

Comment:
Hi!

Is these new types (all three) are correct?
At first look they should be GAUGE, not DERIVE.

DERIVE is intended for reporting of rate of value change, and that is not applicable to allocation/available capacity, I think.

Can you please comment this and (or) update PR if needed?

Thanks.
Sorry for review delay :-)